src/main/java/org/hyperledger/fabric/gateway/Gateway.java

```diff
@@ -132,6 +132,14 @@ static Builder createBuilder() {
          */
         Builder identity(Wallet wallet, String id) throws IOException;
 
+        /**
+         * Specifies the identity that is to be used to connect to the network.  All operations
+         * under this gateway connection will be performed using this identity.
+         * @param identity The Identity using which operations has to be performed.
```

Comment:
The description of the identity parameter doesn't make sense to me. Even something as simple as "An identity" would be clearer to me.